### PR TITLE
Feature/id/9 Admin Panel Panel Backend Integration

### PR DESCRIPTION
**Pull request by victor-villca**
**What did I do?**
I added the ability to save UI form values into a database using an Express.js CRUD API. This involved integrating Axios for frontend-backend communication.

**How did I do it?**
1. Used the CRUD routes in Express.js.
2. Integrated Axios for HTTP requests.
3. Applied the database operations for Creating new products.

**Why did I do it?**
This feature ensures data persistence. It enhances the security and reliability of our application's data storage.

**Author:** @victor-villca 
**Code-Reviewer:** @Elvis-Castro-H 
**Tester:** @saletG08

### CODE REVIEW

**Review by Elvis-Castro-H — APPROVED** [review]
Good job

File: src/app/AddProductForm.js [hunk 1]
```diff
@@ -229,7 +250,7 @@ export default function AddProductForm() {
       <SelectField
         label="Categoria"
         name="categoria"
-        value={formData.categoria}
+        value={formData.category}
         onChange={handleCategoriaChange}
         options={["Seleccione una categoría", "Women", "Men", "Boy", "Girl"]}
```
Comment (saletG08) [inline src/app/AddProductForm.js:255]: Change it to English

Comment (victor-villca) [reply src/app/AddProductForm.js:255]: The suggestion is not possible due to it is already in English, please review the code carefully.

**Review by saletG08 — APPROVED** [review]
I don't have comments

**Pull request merged**